node-red-standalone/umbrel-app.yml

```
@@ -0,0 +1,29 @@
+manifestVersion: 1
+id: node-red-standalone
+category: Automation
+name: "Node-RED (Standalone)"
```

This can just be Node-RED:

```suggestion
name: "Node-RED"
```